models/pipeline.js

```diff
@@ -27,6 +27,11 @@ const MODEL = {
         .description('Unique identifier for the application')
         .example('github.com:123456:master'),
```

Comment:
Yes, we can use `scmUri` instead of `scmContext`, but I think that `scmContext` is effective when we want to filter pipelines by the scm  by which user log-ined.